includes/gateways/paypal-standard.php

```diff
@@ -457,6 +457,19 @@ function edd_process_paypal_ipn() {
  * @return void
  */
 function edd_process_paypal_web_accept_and_cart( $data, $payment_id ) {
+
+	/**
+	 * PayPal Web Accept Data
+	 *
+	 * Allows filtering the Web Accept data that PayPal passes back in via IPN with PayPal Standard
+	 *
+	 * @since 3.8.13
+	 *
+	 * @param array $data      The PayPal Web Accept Data
+	 * @param int  $payment_id The Payment ID associated with this IPN request
+	 */
+	$data = apply_filters( 'edd_paypal_web_accept_and_cart_data', $data, $payment_id );
```

Comment:
Could we change this so that it's a filter available to all IPN types instead of just web_accept and cart_data?

Reply:
@pippinsplugins yeah we can add more than this one filter. I'll add a few more for the paypal standard IPN responses.